tests/kuttl/tests/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: designate.openstack.org/v1beta1
+kind: DesignateAPI
+metadata:
+  name: designate
+spec:
+  databaseInstance: openstack
+  databaseUser: designate
+  serviceUser: designate
+  containerImage: quay.io/tripleowallabycentos9/openstack-designate-api:current-tripleo
```

Comment:
IIUC there is an initiative to switch to tcib images. Shouldn't we change the kuttl tests to use them as well? (see https://github.com/openstack-k8s-operators/designate-operator/pull/44)

Reply:
I don't see one on quay for designate.  Does it get created when it runs the first time?

Reply:
I updated this PR anyhow.

Reply:
thanks, yeah i see it now, must have been more pages :-)